lib/smart_answer/calculators/holiday_entitlement.rb

```diff
@@ -84,8 +95,28 @@ def fraction_of_year
       end
     end
 
+    def shift_entitlement
+      minimum_days_per_week = [shifts_per_week, MAXIMUM_STATUTORY_DAYS_PER_WEEK].min
+
+      if left_before_year_end? || worked_partial_year?
+        format_number(calculate_holiday * minimum_days_per_week, 2)
+      elsif start_date.present? && leave_year_start_date.present?
+        ((calculate_holiday * minimum_days_per_week * 2).ceil / 2.00).to_s
+      else
+        format_number((calculate_holiday * shifts_per_week))
```

Comment:
Using
```ruby
format_number((calculate_holiday * shifts_per_week))
```
results in rounding up this to one decimal place:
```ruby
5.6 * fraction_of_year * shifts_per_shift_pattern / days_per_shift_pattern * 7
```
but the formula we have in the spreadsheet rounds up this to one decimal place:
```ruby
5.6 * shifts_per_shift_pattern / days_per_shift_pattern * 7
```
without `fraction_of_year`. So I think it should be
```ruby
format_number((STATUTORY_HOLIDAY_ENTITLEMENT_IN_WEEKS * shifts_per_week))
```